core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -700,6 +700,8 @@ private[spark] object SparkConf extends Logging {
     KERBEROS_FILESYSTEMS_TO_ACCESS.key -> Seq(
       AlternateConfig("spark.yarn.access.namenodes", "2.2"),
       AlternateConfig("spark.yarn.access.hadoopFileSystems", "3.0")),
+    EVENT_LOG_ALLOW_EC.key -> Seq(
+      AlternateConfig("spark.eventLog.allowErasureCoding", "3.0")),
```

Comment:
`SPARK-25855` seems fixed as of 3.0. I think we can just rename right away.

Reply:
Oh..I see.... Thank you for reminding it.